README.md

```diff
@@ -61,8 +61,8 @@ dependencies: [
 	.package(url: "https://github.com/adobe/aepsdk-edge-ios.git", from: "1.1.0"),
 targets: [
```

Comment:
i think the `targets` array and `dependencies` array are meant to be on the same level of this block - currently `targets` is nested inside of `dependencies`, and it should be moved out